src/load/argdown.ts

```diff
@@ -0,0 +1,114 @@
+import * as date from "../date.js";
+import * as model from "../model/index.js";
+import * as argdownSchema from "../schemas/argdown.js";
+import {
+  ArgdownApplication,
+  IArgdownRequest,
+  ParserPlugin,
+  ModelPlugin,
+  PreselectionPlugin,
+  MapPlugin,
+  IMapNode,
+  IMapEdge,
+} from "@argdown/core";
+
+export function argdown(obj: string) {
+  const app = new ArgdownApplication();
+
+  const parserPlugin = new ParserPlugin();
+  app.addPlugin(parserPlugin, "parse-input");
+
+  const modelPlugin = new ModelPlugin();
+  app.addPlugin(modelPlugin, "build-model");
+
+  const preselectionPlugin = new PreselectionPlugin();
+  app.addPlugin(preselectionPlugin, "preselection-plugin");
+
+  const mapPlugin = new MapPlugin();
+  app.addPlugin(mapPlugin, "map-plugin");
+
+  const request: IArgdownRequest = {
+    input: obj,
+    process: [
+      "parse-input",
+      "build-model",
+      "preselection-plugin",
+      "map-plugin",
+    ],
+    logLevel: "verbose",
+  };
+  const response = app.run(request);
+
+  // Create and add the atom nodes
+  var nodes: { [key: string]: model.Node } = Object.fromEntries(
+    response
+      .map!.nodes.filter((value) => inEdges(value, response.map!.edges))
```

Comment:
Unpacking nullable values with `!` can lead to runtime errors. I would prefer explicit checks using `if` here.